### PR TITLE
Update messaging-policies-in-teams.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8201
Added note

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/messaging-policies-in-teams.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ Messaging policies are used to control which chat and channel messaging features
 
 Users in your organization will automatically get the global policy, unless you create and assign a custom policy. Edit the settings in the global policy or create and assign one or more custom policies to turn on or turn off the features that you want.
 
+> [!NOTE]
+> To ensure syncing after a policy change, a reboot may be necessary in certain instances.
```
Comment (JohanFreelancer9) [inline Teams/messaging-policies-in-teams.md:34]:
```suggestion
> To ensure syncing after a policy change, a reboot may be necessary for certain instances. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
reviewed + confirmed

**Pull request merged**